rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/QueueAttributesDirectives.scala

```diff
@@ -17,6 +17,9 @@ import scala.concurrent.Future
 trait QueueAttributesDirectives {
   this: ElasticMQDirectives with AttributesModule =>
 
+  val awsRegion: String
```

Comment:
abstrakcyjne wartości lepiej robić w scali jako `def`y: zostawia to możliwość decyzji o implementacji jako `val`/`lazy val`/`def` przez implementującego, i jest bardzije odporne na problemy z kolejnością inicjalizacji